browser/src/map/Clipboard.js

```diff
@@ -901,15 +901,20 @@ L.Clipboard = L.Class.extend({
 			try {
 				await clipboard.write([clipboardItem]);
 			} catch (error) {
-				window.app.console.log('navigator.clipboard.write() failed: ' + error.message);
-
-				// Warn that the copy failed.
-				this._warnCopyPaste();
-				// Once broken, always broken.
-				L.Browser.clipboardApiAvailable = false;
-				window.prefs.set('clipboardApiAvailable', false);
-				// Prefetch selection, so next time copy will work with the keyboard.
-				app.socket.sendMessage('gettextselection mimetype=text/html,text/plain;charset=utf-8');
+				// "Document is not focused." is an error but it shouldn't stop the usage of clipboard API.
+				if (error.message.includes('Document is not focused.')) {
```

Comment:
hmm, it seems to be DOMException: https://developer.mozilla.org/en-US/docs/Web/API/Clipboard/write#notallowederror

I assume thete is only that generic code available, and no other "identifier". We have to compare message?

Reply:
Did you check different browsers? Do they have the same message? or it is some specific case?

Reply:
or maybe we can detect `lack of focus` ?

Reply:
Yep, checking if document has focus is a better way. Thanks for checking.